test/e2e/e2e.go

```diff
@@ -306,6 +342,29 @@ func CreatePod(clients *test.Clients, pod *corev1.Pod, _ logging.FormatLogger, c
 	return nil
 }
 
+// SendFakeEventToChannel will create fake CloudEvent and send it to the given channel
```

Comment:
What's fake about this event?

It seems pretty heavy-handed to spin up a pod just to send a single event, can't we send it from this test client?

Reply:
Currently we'll have to create a pod for sending events, since the tests run outside of the cluster and they do not have access to send events directly to the channel. 
More information can be found in the previous discussion:
https://github.com/knative/eventing/pull/726#discussion_r248113303
https://github.com/knative/eventing/pull/726#discussion_r248113386